app/services/data_migrations/backfill_current_course_option_id.rb

```diff
@@ -0,0 +1,17 @@
+module DataMigrations
+  class BackfillCurrentCourseOptionId
+    TIMESTAMP = 20210419154319
+    MANUAL_RUN = false
+
+    def change
+      sql = <<~POPULATE_CURRENT_OPTION_ID_SQL.squish
+        UPDATE application_choices
+           SET current_course_option_id = COALESCE(current_course_option_id, course_option_id)
```

Comment:
Should this also consider `offered_course_option_id`?

Reply:
That's what it should say! Fixing...